LibraBFT/Impl/OBM/Logging/Logging.agda

```diff
@@ -28,3 +28,6 @@ withErrCtxt = id
 
 logEE : ∀ {A} → LBFT A → LBFT A
 logEE f = logInfo >> f >>= λ r → logInfo >> pure r
+
+lcheck : Bool → ErrLog → Either ErrLog Unit
```

Comment:
In #74, see [here](https://github.com/oracle/bft-consensus-agda/blob/mark-verifyqcm-plus-more-impl/LibraBFT/Impl/OBM/Logging/Logging.agda#L32), I have defined this as:
```
lcheck : ∀ {ℓ} {A : Set ℓ} ⦃ _ : ToBool A ⦄ → A → {- List String → -} Either ErrLog Unit
lcheck b {- t -} = case check (toBool b) [] of λ where
  (Left e)  → Left fakeErr
  (Right r) → Right r
```
I tried using your version to avoid conflicts, but it didn't work for all cases, I think because of the lack of instance argument for `toBool`.

Reply:
I updated this PR to include your `Prelude.check` and your `Logging.lcheck`.

However, I updated `lcheck` so uncomments and uses it `List String` argument (since `check` uses it).